app/assets/stylesheets/components/_organizations.scss

```diff
@@ -1,30 +1,99 @@
-.organization-details {
-  .section {
-    border-top: $base-border;
-    margin-top: $base-spacing;
-    padding-top: $base-spacing / 2;
+$actions-height: 3 * $base-spacing;
 
-    &.location {
-      border: $base-border;
-      padding: $base-spacing / 2;
+.organization {
+  &-details {
+    .section {
+      border-top: $base-border;
+      margin-top: $base-spacing;
+      padding-top: $base-spacing / 2;
+
+      &.location {
+        border: $base-border;
+        padding: $base-spacing / 2;
+      }
+    }
+
+    .subsection {
+      margin-top: $base-spacing;
+    }
+
+    .column {
+      @include span-columns(4.5 of 9);
+
+      &:last-child {
+        @include omega;
+      }
+    }
+
+    .map {
+      @include fill-parent;
+      height: 20em;
+      margin-bottom: $base-spacing;
     }
   }
 
-  .subsection {
-    margin-top: $base-spacing;
+  &-modal {
+    @include padding($base-spacing);
+    background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.